src/api/reports/rawcsv.py

```diff
@@ -30,6 +30,23 @@ def _flatten_record(self, record_dict):
     def _apply_formatters(self, flat_record):
         return self._flatten_record(flat_record)
 
+    def data(self, fields, data, *args, **kwargs):
```

Comment:
I may need to talk this one through... Wondering about 
- removing `self._apply_formatters()` from `stream`
- renaming `self.data()` to something like `self.formatted_data` (to separate out the formatting, allowing one in theory to be able to stream non-formatted data/headers)
- removing `self.stream_list` altogether, and adjusting `self.stream` header writing as necessary
but I'll bet there are reasons not to use this approach. Maybe it's a question of whether we need to preserve `self.stream` at all?